frontend/src/app/organization/organization.service.ts

```diff
@@ -55,4 +59,9 @@ export class OrganizationService {
     return this.http.get<Organization[]>(this.url)
       .subscribe(organizations => this.organizationsEvent.next(organizations));
   }
+
+  getOffersForOrganization(id: number) {
+    return this.http.get<Offer[]>(`${this.url}/${id}/offers/`)
+      .subscribe(offers => this.offersEvent.next(offers));
```

Comment:
I guess we should add default image to offers before sending them to the event. The same as it is done in offers.service

Reply:
As a bonus, instead of doing it 'copy-paste' style, I would make the offer.utils.ts file next to the offers.service.ts file, move the "loadDefaultImage" function there and make it exportable, then import it here and to offers.service. Or somebody can do it maybe later someday ;)